ci/jenkins/test-mc.sh

```diff
@@ -220,8 +220,7 @@ function deliver_multicluster_controller {
     export PATH=${GOROOT}/bin:$PATH
 
     docker images | grep 'mc-controller' | awk '{print $3}' | xargs -r docker rmi || true
-    cd multicluster && mkdir -p bin && ln -s /usr/local/bin/kustomize ./bin/kustomize || true
-    make docker-build && cd ..
+    export NO_PULL=1;make antrea-mc-controller
```

Comment:
Is the export here necessary? I assume it is only for the context of the makefile.

Reply:
yes, it will skip pulling image from docker hub which can help to avoid below issue, since we will build local image anyway, so no need to pull the old image first.
```
toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```

Reply:
Will `NO_PULL=1 make antrea-mc-controller` do the trick? I assume we do not need to export the variable.

Reply:
we need it to skip the option `--pull`, here is the definition https://github.com/antrea-io/antrea/blob/main/Makefile#L382-L386, it will try to pull newer version of the image first when it's with '--pull' option.

Reply:
I mean, it seems we can achieve the same result by not exporting the variable by `NO_PULL=1 make antrea-mc-controller`. This limits the scope of the `NO_PULL` variable to the context of the `make` sub-processes only.

Reply:
just sync with @xliuxu  offline, I misunderstood his point, since both ways works as blow,I will keep as it is. thanks.
```
NO_PULL=1 make antrea-mc-controller
export NO_PULL=1;make antrea-mc-controller
```